mcda/mcda_functions/aggregation.py

```diff
@@ -30,32 +26,31 @@ def __init__(self, weights: list):
         if sum(self.weights) != 1:
             self.weights = [val / sum(self.weights) for val in self.weights]
 
-    def weighted_sum(self, norm_indicators: pd.DataFrame()) -> pd.Series(dtype='object'):
+    def weighted_sum(self, norm_indicators: pd.DataFrame) -> pd.Series(dtype='object'):
         """
-        Weighted-sum or additive aggregation function
-        gets as input the normalized values of the indicators in a matrix
-        and estimates the scores over the indicators, per alternative.
+        Weighted-sum or additive aggregation function gets as input the normalized values of the indicators in a matrix
+        and estimates the scores over the indicators, per alternative. The norm_indicators has
+        shape = (no.alternatives x num. indicators) and the returned scores has length = num. of alternatives.
```

Comment:
`no.`?
`num. indicators`? is space indented? Either something similar to `length = num. of alternatives.`?